distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestSecurity.java

```diff
@@ -230,6 +230,28 @@ public void testBasicRestAccess() throws Exception {
     }
 
     @Test
+    public void testSamlFederatedAuth() throws Exception {
+        String recordId = TestCatalog.ingest(Library.getSimpleGeoJson(), "application/json");
+        configureRestForBasic();
+
+        // Creating a new OpenSearch source with no username/password.
+        // When an OpenSearch source attempts to authenticate without a username/password it will
+        // use the subject in the request to create a SAML authentication token
+        OpenSearchSourceProperties openSearchProperties = new OpenSearchSourceProperties(
+                OPENSEARCH_SOURCE_ID);
+        createManagedService(OpenSearchSourceProperties.FACTORY_PID, openSearchProperties);
+
+        waitForFederatedSource(OPENSEARCH_SOURCE_ID);
+
+        String openSearchQuery = SERVICE_ROOT + "/catalog/query?q=*&src=" + OPENSEARCH_SOURCE_ID;
+        given().auth().basic("admin", "admin").when().get(openSearchQuery).then().log().all()
+                .assertThat().statusCode(equalTo(200)).assertThat().body(containsString("myTitle"));
+
+        configureRestForAnonymous();
```

Comment:
Except each test will have its own ingest needs and therefore we won't be able to do the ingest globally in a `@BeforeTest`. And then each test is the only one that will have the ids of the ingested metacards in order to delete them to clean up the state. I suppose we could have the `@BeforeTest` method or an `@AfterTest` method (I think that exists...) do a mass deletion of all metacards but I'm not sure that's the right approach.

And the metacard deletion may or may not require the security to be reconfigured for anon first, so even setting that in a meta method isn't really that helpful.